src/Domain/Creators/CylindricalBinaryCompactObject.cpp

```diff
@@ -63,6 +63,7 @@ CylindricalBinaryCompactObject::CylindricalBinaryCompactObject(
     typename IncludeInnerSphereB::type include_inner_sphere_B,
     typename IncludeOuterSphere::type include_outer_sphere,
     typename OuterRadius::type outer_radius,
+    typename UseEquiangularMap::type use_equiangular_map,
```

Comment:
Just write `bool`, no need to hide the type. We've removed this unnecessary obscuration in the other domain creators too (except for complicated `std::variant` types). A separate commit is fine.